src/pipeline.ts

```diff
@@ -276,7 +276,7 @@ async function findSearchableTorrents() {
 			.map((t) => t.unwrapOrThrow());
 	} else {
 		if (typeof torrentDir === "string") {
```

Comment:
I'm not sure it's not a ridiculous thought, but maybe checking for length to make sure someone doesn't think an empty string is the same as disabling?

I know that seems extreme, but I've seen it in support. This can either be done in Zod or here, I'm not sure transforming for them and allowing the presumption that its a valid config is right. I'll yield to you to make that call if you think its a worthwhile addition.

Reply:
I just realized, this might be addressed already with verifyPath....just need to confirm and other than that it lgtm

Reply:
it ENOENTs

Reply:
Should we be accounting for this with a transform in Zod or what, if it ENOENT's wouldn't that fail to start?

Reply:
I think it's covered by verifyPath.

Reply:
What would we transform it to?

Reply:
i guess if its empty null/undefined?

Reply:
I think I'd rather it just be an error. An empty string is most likely a mistake, which throwing an error surfaces.